### DISCUSSION TITLE
How to Delete Applications and Endpoints from UI

### DISCUSSION THREAD

**Discussion by EGrun** [noop-inc/documentation#140] [Questions] [answered]
How do I delete Applications and Endpoints from a Workshop?

**Comment by 1800joe** [comment, marked as answer]
Applications: 

Go to the application settings page and select delete. See screenshots below:

<img width="436" alt="Screenshot 2024-03-22 at 11 16 20 AM" src="https://github.com/noop-inc/documentation/assets/3111670/0bebb948-ef03-4b5c-aed8-b099aa01c3d6">

<img width="301" alt="Screenshot 2024-03-22 at 11 16 41 AM" src="https://github.com/noop-inc/documentation/assets/3111670/1e5b9e6d-0bee-413c-b904-26171762c609">


Endpoints:

Go to the endpoint page:
<img width="514" alt="Screenshot 2024-03-22 at 11 18 02 AM" src="https://github.com/noop-inc/documentation/assets/3111670/35d18705-eb2b-4250-a554-d6bf09c44477">

Then the Endpoint Settings page:
<img width="417" alt="Screenshot 2024-03-22 at 11 18 20 AM" src="https://github.com/noop-inc/documentation/assets/3111670/89921b05-a629-4687-92cd-6bc73fb81065">

Then same delete button:
<img width="358" alt="Screenshot 2024-03-22 at 11 18 28 AM" src="https://github.com/noop-inc/documentation/assets/3111670/ac5f0c81-b3f1-4211-b4d4-0f538881e8d1">

**Comment by EGrun** [comment]
Is there a configuration file where these settings can be modified?

**Reply by 1800joe** [reply]
Some settings are configurable in the blueprint, is there one in particular you are thinking about?

**Reply by EGrun** [reply]
Specifically, I would be interested in a method to add /remove Applications without using the UI.
